### DISCUSSION TITLE
[Feature] Preload images in slideshow / asset viewer

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#7664] [Feature Request]
### The feature

Currently, when viewing an asset in fullscreen on the web (either by clicking on it or by starting a slideshow), there is a noticeable delay when moving to the next image where a black screen and a blue loading circle are displayed.

Solution: when viewing an image in full screen view, the Immich web viewer should preload the next and previous images to allow for instant switching to next/previous image. The absence of preloaded images is a pretty big barrier to slideshows especially.

When compared to app-that-shall-not-be-named, the difference is pretty obvious. On the other website, if you go really fast you will get blurry previews, but going at a normal speed the fullscreen quality image is already loaded before going to the next image.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Duplicate of #5887 apparently but I really think this is a major QOL issue that seems like it would be easy to add.

**Comment by mmomjian** [comment]
Related request: can show blown up version of webP preview while full version is loading.

**Comment by samholton** [comment]
Preload taken care of in https://github.com/immich-app/immich/pull/7920

Did not address initially showing the preview image large

**Reply by mmomjian** [reply]
Amazing! Thanks for your PR. Surprised you were able to do it with relatively few LOC. I looked at the video examples and I think this will really improve the Slideshow experience.
